### PR TITLE
[MKM] Implement Pyrotechnic Performer

### PR DESCRIPTION
**Pull request by Cguy7777**

### CODE REVIEW

File: Mage.Sets/src/mage/sets/MurdersAtKarlovManor.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
  */
 public final class MurdersAtKarlovManor extends ExpansionSet {
 
-    private static final List<String> unfinished = Arrays.asList("Aurelia's Vindicator", "Bubble Smuggler", "Coveted Falcon", "Crowd-Control Warden", "Expose the Culprit", "Fugitive Codebreaker", "Lumbering Laundry", "Museum Nightwatch", "Pyrotechnic Performer", "Unyielding Gatekeeper");
+    private static final List<String> unfinished = Arrays.asList("Aurelia's Vindicator", "Bubble Smuggler", "Coveted Falcon", "Crowd-Control Warden", "Expose the Culprit", "Fugitive Codebreaker", "Lumbering Laundry", "Museum Nightwatch", "Unyielding Gatekeeper");
```
Comment (xenohedron) [inline Mage.Sets/src/mage/sets/MurdersAtKarlovManor.java:15]: hm, not specifically related to this PR, but I'm pretty sure this unfinished can be removed now that Disguise is implemented. The reason for it is when cards are being partially implemented (missing an unimplemented keyword mechanic but with all other code present).

Comment (JayDi85) [reply Mage.Sets/src/mage/sets/MurdersAtKarlovManor.java:15]: There are must be only 3 cards with cloak [from MKC](https://scryfall.com/search?q=oracle%3Acloak+set%3Amkc&unique=cards&as=grid&order=name) and 5 cards [from MKM](https://scryfall.com/search?q=oracle%3Acloak+set%3Amkm&unique=cards&as=grid&order=name)

**Comment by Cguy7777** [comment]
As suggested [here in #11901](https://github.com/magefree/mage/pull/11901#pullrequestreview-1913110123), I looked into using DamagePlayersEffect instead of a custom effect. The issue is that that effect always sets the source of the damage as the ability's source, which won't be correct when another creature is turned face up.

File: Mage.Sets/src/mage/cards/p/PyrotechnicPerformer.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(2);
+
+        // Disguise {R}
+        this.addAbility(new DisguiseAbility(this, new ManaCostsImpl<>("{R}")));
+
+        // Whenever Pyrotechnic Performer or another creature you control is turned face up, that creature deals damage equal to its power to each opponent.
+        this.addAbility(new TurnedFaceUpAllTriggeredAbility(
+                new PyrotechnicPerformerEffect(),
+                new FilterControlledCreaturePermanent("{this} or another creature you control"),
```
[the remark this answers is not in the document]
Comment (Cguy7777) [reply Mage.Sets/src/mage/cards/p/PyrotechnicPerformer.java]: Good point, thanks. I think I based that off of other cards which were done similarly, probably before `FilterPermanentThisOrAnother` was added; I'll work on a PR to refactor those.

[the remark this answers is not in the document]
Comment (Cguy7777) [reply Mage.Sets/src/mage/cards/p/PyrotechnicPerformer.java]: Good point, thanks. I think I based that off of cards written similarly, likely before `FilterPermanentThisOrAnother` was added. I'll look into those.

**Review by xenohedron — APPROVED** [review]

**Pull request merged**